Native~/Utility.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <string>
+
+std::wstring StrToWstr(const std::string& str)
+{
+    auto length = str.length();
+    std::wstring wstr(length, 0);
+    for (size_t i = 0; i < length; ++i)
+    {
+        wstr[i] = static_cast<std::wstring::value_type>(str[i]);
```

Comment:
Swapping `const char*` for `const LPCWSTR` did the trick. C# should automatically marshall for wide string based on the signature. I'll test with non-Latin characters just in case.